it/common.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+set -exo pipefail
```

Comment:
```suggestion
set -euo pipefail
```

Did you mean `u` (undefined variables) over `x` (debugging output)?

Reply:
Ah, this is my oversight. This line was added for my own testing, and I forgot to remove it when putting out the PR. Though it's my opinion that putting `set -euxo pipefail` in testing scripts might not be a bad idea - but this feels like a separate PR. I *think* setting `-u` in the script will quickly fail the execution (due to some unset variables) in its current state, and therefore I did not have it included during testing.